### PR TITLE
CDN 156 Refactor Window Resizer

### PR DESCRIPTION
**Pull request by MatzElectronics**
**This PR must be merged in conjunction with [solo PR # 36](https://github.com/parallaxinc/solo/pull/36/commits) and [BlocklyProp PR # 1694](https://github.com/parallaxinc/BlocklyProp/pull/1694) repos!**

The code beautifier makes the diff hard to read, but the only changes to the JS are at line 635 (resetToolBoxSizing)

The refactor is based strongly on code recommended by the Google for Education:Blockly developer's site.

### CODE REVIEW

**Comment by zfi** [comment]
This PR also addresses #156, Refactor window resizing.

**Comment by pjewald** [comment]
Note to Self: Disable the CI/CD Demo pipeline while we test this.

File: editor.js [hunk 1]
```diff
@@ -531,24 +531,40 @@ var showNewProjectModal = function(openModal) {
  * @param d milliseconds to delay the resizing, especially if used after a change in the window's location or a reload
  */
 function resetToolBoxSizing(d) {
+    // Vanilla Javascript is used here for speed - jQuery could probably be used, but this is faster.
     if (d) {
```
Comment (pjewald) [inline editor.js]: Let's use descriptive variable names. For example, use delay instead of 'd'.

File: editor.js [hunk 2]
```diff
@@ -531,24 +531,40 @@ var showNewProjectModal = function(openModal) {
...
+                blocklyDiv[i].style.top = navTop + 'px';
+                blocklyDiv[i].style.width = navWidth + 'px';
+                blocklyDiv[i].style.height = navHeight + 'px';
             }
-            //Blockly.mainWorkspace.getCanvas().resize();
+
+            Blockly.svgResize(Blockly.mainWorkspace);
         }, d);
     } else {
-        let navHeight = $(window).height() - $('tr').first().outerHeight();
```
Comment (pjewald) [inline editor.js]: // Resize without delay

**Pull request merged**